src/amo/components/AddonReviewListItem/index.js

```diff
@@ -231,7 +231,12 @@ export class AddonReviewListItemBase extends React.Component<Props> {
         {reviewBody}
         <div className="AddonReviewListItem-byline">
           {review && !isReply ?
-            <Rating styleName="small" rating={review.rating} readOnly />
+            <Rating
+              styleSize="small"
+              review={review}
+              rating={review.rating}
```

Comment:
Now that the component accepts a `review` object, it no longer needs to accept `rating`.